locations/spiders/save_on_foods_ca.py

```diff
@@ -0,0 +1,32 @@
+from scrapy import Spider
+
+from locations.items import Feature
+from locations.pipelines.address_clean_up import clean_address
+
+# SaveOnFoods
+
+
+class SaveOnFoodsCASpider(Spider):
+    name = "Save_On_Foods_ca"
+    item_attributes = {"brand": "Save-On-Foods", "brand_wikidata": "Q7427974"}
+    start_urls = ["https://storefrontgateway.saveonfoods.com/api/stores"]
+
+    def parse(self, response):
+        for store in response.json()["items"]:
+
+            item = Feature()
```

Comment:
One other trick that is a bit of a timesaver is `DictParser`.

Usage example:
https://github.com/alltheplaces/alltheplaces/blob/4cda3aae1f656ff6954acb22e04af96ffa1a12ee/locations/spiders/dm.py#L31

It is clever enough to look at a hash and the most common variants of specific keys, ie `postcode`  or `postal_code` and extract them.

Though it makes your spider slightly less exact, it does save a bunch of repeated code.